astroid/brain/brain_gi.py

```diff
@@ -74,7 +74,9 @@ def _gi_build_stub(parent):
 
         try:
             obj = getattr(parent, name)
-        except AttributeError:
+        except Exception:  # pylint: disable=broad-except
```

Comment:
I assume this can't be tested easily?

Reply:
The entire brain is untested. I can't even install `PyGObject` on any of my machines with `pip`.

Reply:
😅 

Seems like this is a likely candidate for a recent discussion we had about moving non-standard `brains` out of the main `astroid` repository into plugins..
But as long as it works it works I guess.

Reply:
I agree.